src/main/java/me/confuser/banmanager/webenhancer/listeners/ReportListener.java

```diff
@@ -1,82 +1,92 @@
-import me.confuser.banmanager.events.PlayerReportedEvent;
-import me.confuser.banmanager.internal.ormlite.stmt.DeleteBuilder;
-import me.confuser.banmanager.webenhancer.WebEnhancer;
-import me.confuser.banmanager.webenhancer.data.LogData;
-import me.confuser.banmanager.webenhancer.data.PlayerPinData;
-import me.confuser.banmanager.webenhancer.data.ReportLogData;
+import java.sql.SQLException;
+import java.util.Iterator;
+
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
 import org.bukkit.event.player.AsyncPlayerPreLoginEvent;
 
-import java.sql.SQLException;
-import java.util.Iterator;
+import com.j256.ormlite.stmt.DeleteBuilder;
 
-public class ReportListener extends Listeners<WebEnhancer> {
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import me.confuser.banmanager.bukkit.api.events.PlayerReportDeletedEvent;
+import me.confuser.banmanager.bukkit.api.events.PlayerReportedEvent;
+import me.confuser.banmanager.common.api.BmAPI;
+import me.confuser.banmanager.common.data.PlayerData;
+import me.confuser.banmanager.common.data.PlayerReportData;
+import me.confuser.banmanager.webenhancer.WebEnhancer;
+import me.confuser.banmanager.webenhancer.data.LogData;
+import me.confuser.banmanager.webenhancer.data.PlayerPinData;
+import me.confuser.banmanager.webenhancer.data.ReportLogData;
 
-  @EventHandler(ignoreCancelled = true, priority = EventPriority.MONITOR)
-  public void notifyOnReport(PlayerReportedEvent event) {
-    PlayerReportData report = event.getReport();
+@RequiredArgsConstructor
+public class ReportListener implements Listener {
+    @NonNull private final WebEnhancer plugin;
 
-    Iterator<LogData> iterator = plugin.getAppender().getQueue().iterator();
+    @EventHandler(ignoreCancelled = true, priority = EventPriority.MONITOR)
+    public void notifyOnReport(final PlayerReportedEvent event) {
+        PlayerReportData report = event.getReport();
 
-    // Create many-to-many relationship
-    while (iterator.hasNext()) {
-      LogData log = iterator.next();
+        Iterator<LogData> iterator = plugin.getAppender().getQueue().iterator();
 
-      try {
-        plugin.getLogStorage().createIfNotExists(log);
-        plugin.getReportLogStorage().create(new ReportLogData(report, log));
-      } catch (SQLException e) {
-        e.printStackTrace();
-      }
+        // Create many-to-many relationship
+        while (iterator.hasNext()) {
+            LogData log = iterator.next();
+
+            try {
+                plugin.getLogStorage().createIfNotExists(log);
+                plugin.getReportLogStorage().create(new ReportLogData(report, log));
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
     }
-  }
 
-  @EventHandler
-  public void reportDeleted(PlayerReportDeletedEvent event) {
-    int id = event.getReport().getId();
+    @EventHandler
```

Comment:
Formatting is a bit strange, can this be updated to use 2 spaces please instead of 4?

Reply:
It's also making it quite difficult to see the actual changes vs format